### PR TITLE
Update message shown when trying to appeal a decision that has already been overturned or affirmed

### LINKED ISSUE #21739: Update error message when appeals are no longer possible
When a reporter tries to appeal a decision but an appeal has already made by another reporter and a decision for that appeal has already been made, we should have a [specific error message](https://docs.google.com/document/d/1NvxF4Zd86AQCFJAtnW7J-YIY_Z9Co05bMfRPTeEg0jw/edit#heading=h.6bmppbyvd58a):

> Thank you for your report.
> 
> We have already reviewed a similar appeal from another reporter, and determined that the previous decision, that this {{content|account}} did not violate Mozilla's policies (https://extensionworkshop.com/documentation/publish/add-on-policies/), was correct.  We will not review duplicate appeals.

### PR DESCRIPTION
**Pull request by diox**
Fixes #21739

### CODE REVIEW

File: src/olympia/abuse/templates/abuse/appeal.html [hunk 1]
```diff
@@ -29,6 +29,10 @@ <h3>{{ _('Decision {0}')|format_html(decision_id) }}</h3>
                 {% endif %}
                 {% if appeal_email_form or appeal_form %}
                     <p> <input id="appeal-submit" type="submit" value="{{ _('OK') }}"> </p>
+                {% elif appealed_decision_already_made %}
+                    <p> {{ _("Thank you for your report.") }} </p>
+                    <p> {{ _("We have already reviewed a similar appeal from another reporter, and have reversed our prior decision. We have taken action against the content and/or account holder in accordance with our policies.") }}</p>
```
Comment (eviljeff) [inline src/olympia/abuse/templates/abuse/appeal.html:34]: Is this always a true statement though?  Sometimes we've reviewed an appeal and decided to uphold the original decision to not take down the content.

Comment (diox) [reply src/olympia/abuse/templates/abuse/appeal.html:34]: Grr, you're right. I specifically asked for an updated copy that could cover that but then copy/pasted the updated copy, forgetting that scenario.

Comment (diox) [reply src/olympia/abuse/templates/abuse/appeal.html:34]: The doc has a separate copy for this so I'll update the PR.

Comment (diox) [reply src/olympia/abuse/templates/abuse/appeal.html]: https://github.com/mozilla/addons-server/pull/21788/commits/c0c29e1c8568d0e6427c7f05eda8990d4ed2b6f1

**Review by eviljeff — APPROVED** [review]

**Pull request merged**